### PR TITLE
Updated to work with GuideConverter in its own repo

### PR DESCRIPTION
**Pull request by jakub-pomykala**

### CODE REVIEW

File: .github/workflows/GuideConverterAction.yml [hunk 1]
```diff
@@ -21,7 +21,6 @@ jobs:
         run: |
           echo "${{ github.event.inputs.branch }}"
           echo "${{ github.event.inputs.guide_name }}"
-
```
Comment (austin0) [inline .github/workflows/GuideConverterAction.yml:24]: Removed line by accident?

File: .github/workflows/GuideConverterAction.yml [hunk 2]
```diff
@@ -31,6 +30,13 @@ jobs:
 
       # Any prerequisite steps
       - uses: actions/checkout@master
+        
+
```
Comment (austin0) [inline .github/workflows/GuideConverterAction.yml:34]: Added 2 lines instead of one?

File: .github/workflows/GuideConverterAction.yml [hunk 3]
```diff
@@ -73,6 +81,14 @@ jobs:
 
       # Any prerequisite steps
       - uses: actions/checkout@master
+      
+
```
Comment (austin0) [inline .github/workflows/GuideConverterAction.yml:85]: Added 2 lines here and line 91/92.

**Review by austin0 — CHANGES_REQUESTED** [review]
Few small changes.

File: .github/workflows/GuideConverterAction.yml (hunk 1 above)
Comment (jakub-pomykala) [reply .github/workflows/GuideConverterAction.yml:24]: Yes indeed, good catch!

File: .github/workflows/GuideConverterAction.yml (hunk 2 above)
Comment (jakub-pomykala) [reply .github/workflows/GuideConverterAction.yml:35]: Done

File: .github/workflows/GuideConverterAction.yml (hunk 3 above)
Comment (jakub-pomykala) [reply .github/workflows/GuideConverterAction.yml]: Done

**Comment by austin0** [comment]
Can you update https://github.com/OpenLiberty/quick-labs/blob/master/.github/workflows/DraftGuideAction.yml at the same time?

**Review by austin0 — APPROVED** [review]
LGTM

**Pull request merged**